thewalrus/csamples.py

```diff
@@ -134,7 +134,7 @@ def generate_thermal_samples(ls, O, num_samples=1):
     return [
         np.random.poisson(
             np.abs(
-                O @ (np.random.normal(0, np.sqrt(rs))) + 1j * np.random.normal(0, np.sqrt(rs))
+                O @ (np.random.normal(0, np.sqrt(rs)) + 1j * np.random.normal(0, np.sqrt(rs)))
```

Comment:
Was this a typo in the original test? How come it passed in the first PR?

Reply:
This was a typo in the original *sampler*. The test were not detecting the issue because they were all single mode tests.